app/services/FileTransferService.scala

```diff
@@ -72,8 +75,10 @@ class FileTransferService @Inject()(
   //    )
```

Comment:
are these serving some purpose, can't we delete these commented out bits

Reply:
this was an actor-based implementation written by Mac, but it wasn't any better than the existing one